root/layout/components/sidebar/AttendanceLinks.js

```diff
@@ -25,28 +25,32 @@ const AttendanceLinks = ({$c, event}: Props) => {
     return null;
   }
   return (
-    <>
-      <h2 className="attendance">
-        {l('Attendance')}
-      </h2>
-      <CollectionList
-        addText={l('Add to a new list')}
-        entity={event}
-        noneText={l('You have no attendance lists!')}
-        usersLink={
-          <EntityLink
-            content={texp.ln(
-              'Found in {num} attendance list',
-              'Found in {num} attendance lists',
-              allCollections.length,
-              {num: allCollections.length},
-            )}
-            entity={event}
-            subPath="attendance"
-          />
-        }
-      />
-    </>
+    <CollectionList
+      addCollectionText={l('Add to a new list')}
+      collaborativeCollections={$c.stash.collaborator_collections}
+      collaborativeCollectionsHeader={l('Collaborative lists')}
+      collaborativeCollectionsNoneText={
+        l('Not collaborating in any attendance lists!')
```

Comment:
What does collaborating in an attendance list mean? Isn’t that something personal?

Reply:
Whatever the users want. I assume it's not common, but you could have a bunch of friends who pick concerts to go to as a group or something.

Reply:
Until now the number of attendance lists used to match the number of MusicBrainz editors to attend this event. Maybe users want to know the number of editors attending to the event rather than the number of attendance lists?

Reply:
There's no real way of knowing though - that a list has 3 collaborators does not guarantee us that all 3 are really planning to attend.

Reply:
It is a bit confusing that attendance lists don’t have the same meaning for everyone.

By the way, I just found another (preexisting) issue: all event collections are listed as attendance lists whereas “Attending” and “Maybe attending” are just subtypes of event collections.

(Collections were not even listed at all before [React conversion of event sidebar](https://github.com/metabrainz/musicbrainz-server/commit/87d726642ca812b6712b68c6c0394fd49af264af).)